lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -2030,7 +2032,8 @@ class CommandObjectTargetModulesDumpSymtab
           }
           if (INTERRUPT_REQUESTED(GetDebugger(),
                                   "Interrupted in dump all symtabs with {0} "
-                                  "of {1} dumped.", num_dumped, num_modules))
+                                  "of {1} dumped.",
```

Comment:
It looks like there are some formatting changes here that are not related to the functional change. We should split those out into a separate NFC PR if we want to include them.

It looks like all the `INTERRRUPT_REQUESTED` changes fall into that category.